server/rest_slicer_cli.py

```diff
@@ -867,24 +1019,25 @@ def genRESTEndPointsForSlicerCLIsInDocker(info, restResource, dockerImages):
     for dimg in dockerImages:
 
         # get CLI list
-        cliListSpec = subprocess.check_output(['docker', 'run',
-                                               dimg, '--list_cli'])
+        cliListSpec = dockerCache.getCLIDict(dimg)
```

Comment:
rename to `getCLIList`? 

Implementing it as a Dict was a convenience to make it easy to find CLIs. I went back and forth between making it a list of dicts vs a dict.